app/subscriber/src/store/hooks/admin/useAlerts.ts

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { useAjaxWrapper, useLookup } from 'store/hooks';
+import { IAdminState, useAdminStore } from 'store/slices';
+import { IAlertModel, useApiAdminAlerts } from 'tno-core';
+
+interface ITagController {
+  findAllAlerts: () => Promise<IAlertModel[]>;
+  getAlert: (id: number) => Promise<IAlertModel>;
+  addAlert: (model: IAlertModel) => Promise<IAlertModel>;
+  updateAlert: (model: IAlertModel) => Promise<IAlertModel>;
+  deleteAlert: (model: IAlertModel) => Promise<IAlertModel>;
+}
+
+export const useAlerts = (): [IAdminState, ITagController] => {
+  const api = useApiAdminAlerts();
```

Comment:
Don't use the admin hook for the subscriber.  It will cause security and maintenance issues going forward.